src/electron-main.js

```diff
@@ -207,6 +207,14 @@ let eventIndex = null;
 let mainWindow = null;
 global.appQuitting = false;
 
+
+const deleteFolderRecursive = async (p) => {
```

Comment:
'deleteContents' might be a better name for this, as it's not actually recursive (alternatively, `fs.unlink` does have a recursive option now, although it is unstable).

Reply:
I saw the `unlink` thing and saw that it was unstable, thus I was sad.

Anyways renamed.